README.md

```diff
@@ -1,47 +1,38 @@
-Hello!
+# 📦 Cranelift Jit Demo
 
 This is a simple demo that JIT-compiles a toy language, using Cranelift.
 
-It uses the new JIT interface in development
-[here](https://github.com/bytecodealliance/wasmtime/tree/main/cranelift/jit). JIT takes care
-of managing a symbol table, allocating memory, and performing relocations, offering
-a relatively simple API.
+It uses the new JIT interface in
+development [here](https://github.com/bytecodealliance/wasmtime/tree/main/cranelift/jit) . JIT takes care of managing a
+symbol table, allocating memory, and performing relocations, offering a relatively simple API.
 
 This is inspired in part by Ulysse Carion's
-[llvm-rust-getting-started](https://github.com/ucarion/llvm-rust-getting-started)
-and Jonathan Turner's [rustyjit](https://github.com/jonathandturner/rustyjit).
-
-A quick introduction to Cranelift: Cranelift is a compiler backend. It's
-light-weight, supports `no_std` mode, doesn't use floating-point itself,
-and it makes efficient use of memory.
-
-And Cranelift is being architected to allow flexibility in how one uses it.
-Sometimes that flexibility can be a burden, which we've recently started to
-address in a new set of crates, `cranelift-module`, `cranelift-jit`, and
-`cranelift-faerie`, which put the pieces together in some easy-to-use
-configurations for working with multiple functions at once. `cranelift-module`
-is a common interface for working with multiple functions and data interfaces
-at once. This interface can sit on top of `cranelift-jit`, which writes
-code and data to memory where they can be executed and accessed. And, it can
-sit on top of `cranelift-faerie`, which writes code and data to native .o files
-which can be linked into native executables.
+[llvm-rust-getting-started](https://github.com/ucarion/llvm-rust-getting-started) and Jonathan
+Turner's [rustyjit](https://github.com/jonathandturner/rustyjit).
+
+A quick introduction to Cranelift: Cranelift is a compiler backend. It's light-weight, supports `no_std` mode, doesn't
+use floating-point itself, and it makes efficient use of memory.
+
+And Cranelift is being architected to allow flexibility in how one uses it. Sometimes that flexibility can be a burden,
+which we've recently started to address in a new set of crates, `cranelift-module`, `cranelift-jit`, and
+`cranelift-faerie`, which put the pieces together in some easy-to-use configurations for working with multiple functions
```

Comment:
Can you replace cranelift-faerie with cranelift-object? cranelift-faerie doesn't exist anymore.

Reply:
Okay